frontend/src/pages/ResultsPage/index.tsx

```diff
@@ -40,17 +40,15 @@ export const ResultsPage: FC = () => {
     let shouldUpdate = true
 
     const init = async () => {
-      try {
```

Comment:
what was the issue with using try catch statement?

Reply:
No issue, just improves readability.